src/app.ts

```diff
@@ -73,9 +74,9 @@ createExpressEndpoints(reviewContract, reviewRouter, app, jwtMiddleware);
 
 app.listen(3000, () => {
   console.log("Server On");
-  // schedule.scheduleJob("0 * * * * *", function () {
-  //   dbScheduler();
-  // });
+  schedule.scheduleJob("0 * * * * *", function () {
+    dbScheduler();
+  });
```

Comment:
```suggestion
  schedule.scheduleJob("0 * * * * *", dbScheduler);
```
함수를 인자 없이 그대로 실행한다면, 함수를 그대로 전달하면 가독성을 올릴 수 있어요!